src/plugin/types/Element.ts

```diff
@@ -45,20 +46,19 @@ export type FieldViews =
 export type NonCustomFieldViews =
   | TextFieldView
   | RichTextFieldView
-  | CheckboxFieldView
-  | DropdownFieldView;
```

Comment:
I'm not totally sure why the DropdownFieldView has been removed here - was this intentional?

Reply:
Good spot. I think this was removed when DropdownFieldView still had a generic, but as I've removed the generic, it can now return. Thanks!

Reply:
In fact, it's no longer needed at all as a result of that change – even better.